Formula/cargo-edit.rb

```diff
@@ -15,12 +15,33 @@ class CargoEdit < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "369459c3c1c778113fb1e17b9ede22ae43d0faef13cc73f6cb21413a89af766d"
   end
 
+  depends_on "pkg-config" => :build
   depends_on "libgit2"
   depends_on "openssl@1.1"
   depends_on "rust" # uses `cargo` at runtime
 
   def install
-    system "cargo", "install", *std_cargo_args
+    ENV["OPENSSL_NO_VENDOR"] = "1"
+
+    # Read the default flags from `Cargo.toml` so we can remove the `vendored-libgit2` feature.
+    cargo_toml = (buildpath/"Cargo.toml").read
+    cargo_option_regex = /default\s*=\s*(\[\n.+?\n\])/m
```

Comment:
```suggestion
    cargo_option_regex = /default\s*=\s*(\[.+?\])/m
```

So that it works for single-line arrays like [this](https://github.com/rust-random/rand/blob/0.8.5/Cargo.toml#L30), too? At least, in case this will be copy-pasted to other formulae sometime.